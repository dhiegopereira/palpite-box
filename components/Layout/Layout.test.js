import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '.'


describe('PageTitle Component', () => {
    it('Return title', () => {
        const tree = renderer
            .create(<Layout/>)
            .toJSON();
        expect(tree).toMatchSnapshot();        
    })
})