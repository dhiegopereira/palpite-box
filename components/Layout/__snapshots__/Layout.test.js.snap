// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle Component Return title 1`] = `
<div>
  <div
    className="bg-gray-200 p-4  shadow-md"
  >
    <div
      className="container mx-auto"
    >
      <a
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        <img
          alt="PalpiteBox"
          className="mx-auto"
          src="/img/logo_palpitebox.png"
        />
      </a>
    </div>
  </div>
  <div
    className="container mx-auto"
  />
  <div
    className="bg-gray-500 p-4 my-96 absolute w-full"
  >
    <div
      className="container mx-auto text-center font-bold text-white"
    >
      Projeto desenvolvido por: Diego Pereira /
      <a
        className="link hover:underline"
        href="https://linkedin.com/in/diegopereirati"
        target="_blank"
      >
        Linkedin
      </a>
       /
      <a
        className="hover:underline"
        href="https://github.com/dhiegopereira"
        target="blank"
      >
        Github
      </a>
    </div>
    <div
      className="container mx-auto"
    >
      <img
        alt="PalpiteBox"
        className="mx-auto"
        src="/img/logo_scio.png"
      />
    </div>
  </div>
</div>
`;
